From 685479290becaa801be52538068cbca8d8dd4b09 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 24 Mar 2017 22:52:02 +0000
Subject: [PATCH 09/10] rescue: docs: Note that you can run virt-rescue on
 disks too.

(cherry picked from commit 267569f7ad22ad14d9ef4fa27b038469ae115c45)
---
 rescue/virt-rescue.pod | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rescue/virt-rescue.pod b/rescue/virt-rescue.pod
index dfa74e204..5cfbd6e1c 100644
--- a/rescue/virt-rescue.pod
+++ b/rescue/virt-rescue.pod
@@ -69,7 +69,8 @@ Virt-rescue can be used on I<any> disk image file or device, not just
 a virtual machine.  For example you can use it on a blank file if you
 want to partition that file (although we would recommend using
 L<guestfish(1)> instead as it is more suitable for this purpose).  You
-can even use virt-rescue on things like SD cards.
+can even use virt-rescue on things like USB drives, SD cards and hard
+disks.
 
 You can get virt-rescue to give you scratch disk(s) to play with.
 This is useful for testing out Linux utilities (see I<--scratch>).
-- 
2.13.0

